cmd/sup/main.go

```diff
@@ -246,6 +250,33 @@ func main() {
 		network.Hosts = hosts
 	}
 
+	// --config flag location for ssh_config file
+	if sshConfig != "" {
+		confHosts, err := sshconfig.ParseSSHConfig(sshConfig)
```

Comment:
This hangs with my `~/.ssh/config`. It enters endless loop and eats 100% CPU.

Reply:
I take it back... it started working for some reason when I recreated my ~/.ssh/config. I don't get it.